packages/backend/src/models/SchedulerModel/index.ts

```diff
@@ -211,6 +213,11 @@ export class SchedulerModel {
                     dashboard_uuid: newScheduler.dashboardUuid,
                     updated_at: new Date(),
                     options: newScheduler.options,
+                    filters:
+                        isDashboardScheduler(newScheduler) &&
+                        newScheduler.filters
+                            ? JSON.stringify(newScheduler.filters)
+                            : null,
```

Comment:
What is the reason for the conditions to be different between create and update?

Reply:
When creating, the type includes `dashboardUuid`: https://github.com/lightdash/lightdash/blob/feat/add-filters-to-schedulers/packages/common/src/types/scheduler.ts#L140
When updating, the type does not include `dashboardUuid` so I can't use the type guard :/  see line 151-153 in `scheduler.ts` below on the PR ⬇️